tests/Model/DataObject/ClassDefinitionTest.php

```diff
@@ -66,8 +66,11 @@ public function testInputSetterCode(): void
 */
 public function setInput(?string $input): static
 {
-	$this->input = $input;
+	if ($this->getInput() !== $input) {
+		$this->markFieldDirty("input", true);
 
+		$this->input = $input;
+	}
```

Comment:
Do we still need those changes, now that you have fixed the test in another way?